vpr/src/device/rr_graph_builder_view.h

```diff
@@ -0,0 +1,53 @@
+#ifndef _RR_GRAPH_BUILDER_VIEW_
```

Comment:
Avoid leading undrescores in the include-guards as they are reserved for c/c++ compiler internals.

Reply:
Agree. Updated.